src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/IChannel.kt

```diff
@@ -117,25 +117,33 @@ interface IChannel {
         val acceptAllCerts: Boolean = false,
     )
 
-    enum class SendMode(val handle: Boolean, val mangle: Boolean) {
+    enum class SendMode(val handle: Boolean, val mangle: Boolean, val socketSend: Boolean, val mstoreSend: Boolean) {
         /**
          * Message and its metadata will pass through [IHandler.onOutgoing] and [IMangler.onOutgoing] before send
          */
-        HANDLE_AND_MANGLE(true, true),
+        HANDLE_AND_MANGLE(true, true, true, true),
```

Comment:
I think we should add parameter names here. 4 boolean parameters are really uninformative and it is hard to understand what is what